routes/api/questions/controller.js

```diff
@@ -97,7 +113,7 @@ class QuestionsController extends BaseController {
     //   }
     // })
 
-    const { score, correctlyAnswered, incorrectlyAnswered } = U.getScore(markedChoices, U.parseIntArray(question.correctAnswers), question.choices)
+    const { score, correctlyAnswered, incorrectlyAnswered } = U.getScore(markedChoices, U.parseIntArray(question.correctAnswers), possibleChoices)
```

Comment:
why you changed this?

Reply:
they were both the same stuff, so i figured pass the variable instead. will change